src/components/ButtonPanel.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => {
+  const array = {
+    group1: ['AC', '+/-', '%'],
+    group2: ['7', '8', '9'],
+    group3: ['4', '5', '6'],
+    group4: ['1', '2', '3'],
+    group5: ['.'],
+  };
+
+  const isTrue = true;
+
+  const generateKey = pre => `${pre}_${new Date().getTime()}`;
```

Comment:
**[OPTIONAL]**
You could use packages like `uuid` or `shortid` to achieve this more efficiently.